core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/console/CodeSystemsCommand.java

```diff
@@ -57,6 +59,9 @@
 public final class CodeSystemsCommand extends Command {
 
 	private static final Ordering<CodeSystem> SHORT_NAME_ORDERING = Ordering.natural().onResultOf(CodeSystem::getShortName);
+	
+	private static final String CODE_SYSTEM_INFO_JOIN_DELIMITER = "\n   ";
+	private static final String LOCALES_ADDITIONAL_PROPERTIES_AVAILABLE_UPGRADES_VERSIONS_JOIN_DELIMITER = "\n      ";
```

Comment:
You can also simplify the naming of this constant to something like CODE_SYSTEM_SUBPROPERTY_DELIMITER and the other one could be CODE_SYSTEM_PROPERTY_DELIMITER

Reply:
Done